homeassistant/components/light/piglow.py

```diff
@@ -0,0 +1,86 @@
+"""
+Support for Piglow LED's.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.piglow/
+"""
+import logging
+
+# Import the device class from the component that you want to support
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS,
+    ATTR_RGB_COLOR, SUPPORT_RGB_COLOR,
+    Light)
+
+# Home Assistant depends on 3rd party packages for API specific code.
+REQUIREMENTS = ['piglow==1.2.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_PIGLOW = (SUPPORT_BRIGHTNESS | SUPPORT_RGB_COLOR)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Piglow Light platform."""
+    import piglow
+
+    # Add devices
+    add_devices([PiglowLight(piglow)])
```

Comment:
Would be nice if there would be a way to fail if the communication with the hardware is not possible. Otherwise the users end up with a broken platform.